networkQuality.go

```diff
@@ -543,6 +550,7 @@ timeout:
 					}
 				} else if probeMeasurement.Type == probe.SelfDown || probeMeasurement.Type == probe.SelfUp {
 					selfRtts.AddElement(probeMeasurement.Duration.Seconds())
+					selfRttsQualityAttenuation.AddSample(probeMeasurement.Duration.Seconds())
```

Comment:
Is adding a sample compute intensive? If so, perhaps we only want to add it when the user has the functionality enabled?

Reply:
Not very, at least not with the default level of compression I've used. In the tests I add 160000 samples and the tests run in ~20 milliseconds on my machine. Checking if it's necessary seems like a good idea either way though, so I've added that.